docs/resource/README.md

```diff
@@ -186,11 +186,11 @@ All custom identifiers SHOULD be stable across different versions of an implemen
 | `webjs` | webjs | ![Stable](https://img.shields.io/badge/-stable-lightgreen) |
 <!-- endsemconv -->
 
-## Telemetry SDK (Experimental)
+## Telemetry Distribution (Experimental)
 
 **Status**: [Experimental][DocumentStatus]
 
-**type:** `telemetry.sdk`
+**type:** `telemetry.distro`
```

Comment:
not related to this PR, feel free to resolve if necessary - I wonder what type means in scope of this document,

Reply:
> I wonder what type means in scope of this document.

Good question. I'm not sure, but I'd guess it means that resource attributes of the same type should be provided by the same resource detector?

Reply:
No idea what the type here is about, seems totally made up 🤔.

Reply:
Created https://github.com/open-telemetry/semantic-conventions/issues/929 to follow up